### ISSUE TITLE
Black screen on Quest 2 when connecting to Windows 10

### ISSUE THREAD

**Issue by TheAquaSheep** [alvr-org/ALVR#1754]
## Description

When connecting from Windows 10 using either a USB connection or wireless, only a purely black screen is being rendered on the headset.
The SteamVR indicator window on the desktop shows the Steam Home room and tracking appears to be sometimes working but mostly not.
The portable version has the same issue.

This is the first version attempted, though it works using Linux on the same PC.

## General Troubleshooting
- [x] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [x] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/alvr-org/ALVR/wiki)

### Hardware
**Note**: for Linux, an upload to the [`hw-probe`](https://linux-hardware.org/) database is preferred: `hw-probe -all -upload`

**CPU**:
Intel i5 8500

**GPU**:
NVIDIA Gefore GTX 1060 6GB
**GPU Driver Version**:
536.40

**Audio**:
Realtek

### Installation
**ALVR Version**:
20.1.0 (PC) / 20.1.0 (Quest 2)
**ALVR Settings File**:
[session.json.txt](https://github.com/alvr-org/ALVR/files/12136464/session.json.txt)
NB/ I added the .txt extension for github to allow the above file. It is not there in the original.

**SteamVR Version**:
1.26.6

**Install Type**:
- [x] Packaged (`exe`, `deb`, `rpm`, etc)
- [x] Portable (`zip`)
- [ ] Source

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):
Windows 10 Pro 21H2 Build 19044.3208

**Comment by HajimeYoshida-lw05006** [comment]
ALVR 20.0.0 or later is a quest installer (alvr_client_quest.apk
) is not available.
Installer for android (alvr_client_android.apk
) on quest2 doesn't seem to work.

An installer for quest2 is available for 19.1.1, so use that and it should work.
However, if you apply the quest's firmware v55, it will crash immediately after starting ALVR in the quest, so don't upload to v54 or higher.

**Comment by TheAquaSheep** [comment]
I have already updated to firmware v55 I believe, based on Runtime version 55.0.0. ... in the system information on the device.
I had no issues with installing the alvr_client_android.apk on the Quest 2, however.
I downloaded the v20.1.0 apk file from the github release page and used `adb install ...` to install it.
It works using the server on Manjaro Linux, although it is a bit jittery.
The connection is established on Windows as the server software states it is streaming and the client software begins receiving the video, but it is completey blank on the client device.
Thanks for the info. Given that you have said it crashes on firmware v55, I will not try the alvr_client_quest.apk from a previous ALVR version.
